bxin/ptg2m2/m2_diagnostic.ipynb

```diff
@@ -73,17 +111,101 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "state = await m2.evt_summaryState.aget(timeout=5)\n",
-    "print('staring with: m2 state', salobj.State(state.summaryState), pd.to_datetime(state.private_sndStamp, unit='s'))\n",
-    "if state.summaryState == 2:\n",
-    "    await salobj.set_summary_state(m2, salobj.State.DISABLED) #disable m2\n",
-    "    await asyncio.sleep(3)\n",
-    "    state = await m2.evt_summaryState.aget(timeout=5)\n",
-    "    print('m2 state', salobj.State(state.summaryState), pd.to_datetime(state.private_sndStamp, unit='s'))\n",
+    "if summit:\n",
+    "    await m2.cmd_enterControl.set_start() "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "sstate = await m2.evt_summaryState.aget(timeout=5)\n",
```

Comment:
`sstate` or `state`?

Reply:
sstate is for summary state.
dstate is for detailed state.
Not the best names. this is in my personal area anyway...